fix(MessageBox): Adjust spacing (#323)
patternfly/virtual-assistant: packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/UI/UI.md

**Note**: When messages update, it is important to announce new messages to users of assistive technology. To do this, make sure to set the `announcement` prop on `<MessageBox>` whenever you display a new message in `<MessageBox>`. You can view this in action in our [basic chatbot](/patternfly-ai/chatbot/overview/demo#basic-chatbot) and [embedded chatbot](/patternfly-ai/chatbot/overview/demo#embedded-chatbot) demos.

### Welcome message

> To introduce users to the chatbot experience, a welcome prompt can fill the message box before they input their first message. This brief message should follow our [conversation design guidelines](/patternfly-ai/conversation-design) to welcome users to the chatbot experience and encourage them to interact.
> 
> The welcome prompt can be dismissed once a user sends their first message. To reposition the direction of the welcome content, you can change the `position` of the `<MessageBox>` component.

Reviewer: ```suggestion
To introduce users to the chatbot experience, display a welcome message before they input their first message. This brief message should follow our [conversation design guidelines](/patternfly-ai/conversation-design) to welcome users to the chatbot experience and encourage them to interact.

This message can be dismissed once a user sends their first message. To change the arrangement of the message within the message box, specify the `position` in the `<MessageBox>` component.
```

Author: Works for me @edonehoo! Thanks.

```

> ### Welcome interaction

Reviewer: Hmm I'm wondering if we could combine these 2 examples, since the demo also shows the prompts? 

Or, we could call this example "welcome message" and move it above the "welcome prompts" example? Then, remove the prompt cards from this demo. That way the focus is on the interaction of hiding the welcome message after a user message, and the welcome prompts" example makes more sense as a standalone.

Wdyt? Are these deviating too far from what you had in mind? Mainly, just wondering if we can streamline since the examples overlap a little 🤔

Reviewer 2: +1 on having a welcome prompts as a standalone

Author: Pushing this now - let me know if you have feelings after it builds @edonehoo and @kaylachumley.

> The welcome prompt can be turned off when a user inputs their first message. You can also reposition the direction of the content via the `position` prop on `MessageBox`.

Reviewer: ```suggestion
The welcome prompt can be dismissed once a user sends their first message. To reposition the direction of the welcome content, you can change the `position` of the `<MessageBox>` component.
```
